docs/about.md

```diff
@@ -0,0 +1,50 @@
+
+## What is OSCAL?
+
+The OSCAL Compass project is set of tools that enable the creation, validation, and governance of documentation artifacts for compliance needs. It leverages NIST's OSCAL (Open Security Controls Assessment Language) as a standard data format for interchange between tools and people, and provides an opinionated approach to OSCAL SDK and adoption by policy engines.
+
+The OSCAL Compass project is hosted by the Cloud Native Computing Foundation [(CNCF)](https://www.cncf.io/)
+
+
+
+
+## Where is OSCAL currently in use?
```

Comment:
Could you share a bit more context around this question? It leaves me a little confused. As an alternative, what do think of bumping `Project Repositories` up a heading level?